src/components/CombinedFeaturePanel/editors/AutoEditor.tsx

```diff
@@ -134,12 +146,55 @@ export const AutoEditor = ({ feature, tagKey }: AutoEditorProps) => {
     });
   }, []);
 
+  const handlePickerValueChange = React.useCallback(
+    (newPickerValue: string) => {
+      const {normalizedTag: baseTag} = normalizeAndExtractLanguageTagsIfPresent(tagName);
+      const updatedTagName = [baseTag, newPickerValue].join(":");
+
+      if (updatedTagName !== finalTagName) {
+        setFinalTagName(updatedTagName);
+
+        const newUrl = router.asPath.replace(
+          `/edit/${tagKey}`,
+          `/edit/${updatedTagName}`,
+        );
+        router.replace(newUrl, undefined, { shallow: true });
+      }
+    },
+    [tagName, tagKey, router, finalTagName],
+  );
+
   const Editor = getEditorForKey(tagKey);
-  if (Editor) {
+  if (Editor === StringFieldEditor) {
+    return (
+      <Editor
+        feature={feature}
+        tagKey={finalTagName}
+        addingNewLanguage={addingNewLanguage}
+        onChange={setEditedTagValue}
+        onUrlMutationSuccess={onUrlMutationSuccess}
+        handleSubmitButtonClick={handleSubmitButtonClick}
+        passLanguagePickerValueToParent={handlePickerValueChange}
+      />
+    );
+  }
+  if (Editor === WheelchairEditor) {
```

Comment:
todo (blocking): This was already removed, might have slipped back in (or I remember wrong?) :)